README.md

```diff
@@ -51,6 +51,8 @@ Kermitigated (twitch handle)
 [Victor Pietro](https://github.com/pietroow) (A.K.A. Pietroow)  
 [Coleyra](https://github.com/Coleyra)  
 [MissingNoShiny](https://github.com/MissingNoShiny)  
+u/Stark_001001 (The Skeld in game map)  
+u/mooseknuckle_king (Mira HQ and Polus in game maps)
```

Comment:
Maybe we should add  their names and link them to their reddit profile.

Reply:
didn't really get their names.. just going off their username on Reddit. I can link their profiles though

Reply:
do you find the text hard to read?

Reply:
Yes, the color is a bit off.